torch/distributed/rpc/api.py

```diff
@@ -357,6 +363,24 @@ def remote(to, func, args=None, kwargs=None):
         >>> import torch.distributed.rpc as rpc
         >>> rpc.init_rpc("worker1", rank=1, world_size=2)
         >>> rpc.shutdown()
+
+        If invoking an annotated TorchScript function, then run the following
+        code in two different processes:
+
+        >>> # On worker 0:
+        >>> @torch.jit.script
+        >>> def my_script_add(t1, t2):
+        >>>    return torch.add(t1, t2)
+        >>> import torch.distributed.rpc as rpc
+        >>> rpc.init_rpc("worker0", rank=0, world_size=2)
+        >>> rref = rpc.remote("worker1", my_script_add, args=(torch.ones(2), 3))
```

Comment:
we also need _remote() version that accepts qualified name only, so that it can be consistent with JIT interface later on. more specifically, the JIT interface will be private temporarily because JIT ops do not accept callable yet, for now we plan to have a private _remote() to accept qualified_name